Flow.Launcher/CustomQueryHotkeySetting.xaml.cs

```diff
@@ -13,14 +13,14 @@ namespace Flow.Launcher
     public partial class CustomQueryHotkeySetting : Window
     {
         private SettingWindow _settingWidow;
+        private readonly Settings _settings;
```

Comment:
any reason to change this?

Reply:
Technically this is a field instead of property (maybe it worth to figure out the difference). For here, I think it's fine.